java/8/Dockerfile

```diff
@@ -25,10 +25,8 @@
 # latest Forge release. This should be removed when the manifest constructor class
 # issue is backported or if Forge decides to abandon old versions and we need
 # newer Java versions. Old Quay images were updated like once a year, so it's ok.
-#
```

Comment:
Please don't remove the comment as it directly points to the issue without having to ask or google around. Other than that, LGTM.